performance-test/scripts/InjiWeb_Test_Script.jmx

```diff
@@ -0,0 +1,2119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
+      <boolProp name="TestPlan.functional_mode">false</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
+      <elementProp name="TestPlan.user_defined_variables" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments"/>
+      </elementProp>
+      <stringProp name="TestPlan.user_define_classpath"></stringProp>
+    </TestPlan>
+    <hashTree>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments">
```

Comment:
Naming convention changes:
Center-MachineIDValues.csv
addIdentityRequestDetails.csv
vid_testdata.txt
authcode_token.txt

Reply:
I have renamed Center-MachineIDValues.csv, addIdentityRequestDetails.csv to center_machine_id_values, add_identity_request_details (as per performance test developer guide names should be small caps seprated by underscore). vid_testdata.txt, authcode_token.txt remains unchanged.